src/ray/raylet/worker_pool.h

```diff
@@ -506,46 +565,10 @@ class WorkerPool : public WorkerPoolInterface, public IOWorkerPoolInterface {
     rpc::RuntimeEnvInfo runtime_env_info;
     /// The dynamic_options.
     std::vector<std::string> dynamic_options;
+    /// The duration to keep the worker alive even if it's idle.
```

Comment:
I think we should make this comment more specific: it's only used by prestart workers and only when the first time it's idle (before running any tasks).

ideally we should also rename the variable name to reflect that.

Reply:
renamed to worker_startup_keep_alive_duration